website/content/docs/developer/first-secret.mdx

```diff
@@ -0,0 +1,392 @@
+---
+layout: docs
+page_title: Your First Secret
+description: Learn how to store and retrieve your first secret.
+---
+
+# Create Your First Secret
```

Comment:
I don't think we should name this "Create Your First Secret" because the retrieving of the secret is technically the more important part of the tutorial and what most devs will be doing (I just had to add the creating part so they wouldn't have to learn how to create secrets with the CLI, which takes some extra configuring/explaining.) 

Maybe just keeping it as "Your First Secret" would be best.